src/services/BackgroundSchedulerService/BackgroundSchedulerService.ts

```diff
@@ -10,33 +14,40 @@ interface PeriodicTask {
 }
 
 // See https://github.com/cds-snc/covid-shield-mobile/issues/642#issuecomment-657783192
-export const DEFERRED_JOB_INTERNVAL_IN_MINUTES = 240;
-const EXACT_JOB_INTERNVAL_IN_MINUTES = 90;
+export const PERIODIC_TASK_INTERVAL_IN_MINUTES = TEST_MODE ? 15 : 240;
+export const PERIODIC_TASK_DELAY_IN_MINUTES = TEST_MODE ? 1 : PERIODIC_TASK_INTERVAL_IN_MINUTES + 5;
 
 const registerPeriodicTask = async (task: PeriodicTask) => {
-  BackgroundFetch.configure(
-    {
-      minimumFetchInterval: TEST_MODE ? EXACT_JOB_INTERNVAL_IN_MINUTES : DEFERRED_JOB_INTERNVAL_IN_MINUTES,
-      forceAlarmManager: false,
-      enableHeadless: true,
-      startOnBoot: true,
-      stopOnTerminate: false,
-    },
-    async taskId => {
-      captureMessage('runPeriodicTask', {taskId});
-      try {
-        await task();
-      } catch (error) {
-        captureException('runPeriodicTask', error);
-      }
-      BackgroundFetch.finish(taskId);
-    },
-  );
   if (Platform.OS === 'ios') {
+    BackgroundFetch.configure(
+      {
+        minimumFetchInterval: PERIODIC_TASK_INTERVAL_IN_MINUTES,
+        forceAlarmManager: false,
+        enableHeadless: true,
+        startOnBoot: true,
+        stopOnTerminate: false,
+      },
+      async taskId => {
+        captureMessage('runPeriodicTask', {taskId});
+        try {
+          await task();
+        } catch (error) {
+          captureException('runPeriodicTask', error);
+        }
+        BackgroundFetch.finish(taskId);
+      },
+    );
     const result = await BackgroundFetch.scheduleTask({taskId: BACKGROUND_TASK_ID, delay: 0, periodic: true}).catch(
       () => false,
     );
     captureMessage('registerPeriodicTask', {result});
+  } else {
+    log.debug({category: 'background', payload: {message: 'registerPeriodicTask - Android'}});
```

Comment:
would be better to just use `message`, ie: `log.debug({category: 'background', message: 'registerPeriodicTask - Android'});`